README.md

```diff
@@ -132,22 +132,33 @@ cd build
 For Homebrew on Intel models:
 
 ```console
-Qt5_DIR=/usr/local/opt/qt5/ cmake -DQT_QMAKE_EXECUTABLE=/usr/local/opt/qt5/bin/qmake ../ImageLounge/.
+# qt5
+$ Qt5_DIR=/usr/local/opt/qt5/ cmake -D QT_VERSION_MAJOR=5 --install-prefix /Applications ../ImageLounge/.
+
+# qt6
+$ cmake -D QT_VERSION_MAJOR=6 --install-prefix /Applications ../ImageLounge/.
 ```
 
 For Homebrew on Apple Silicon models:
 
 ```console
-Qt5_DIR=/opt/homebrew/opt/qt5/ cmake -DQT_QMAKE_EXECUTABLE=/opt/homebrew/opt/qt5/bin/qmake ../ImageLounge/.
+$ Qt5_DIR=/opt/homebrew/opt/qt5/ cmake -DQT_QMAKE_EXECUTABLE=/opt/homebrew/opt/qt5/bin/qmake --install-prefix /Applications ../ImageLounge/.
 ```
 
 Run make:
 
 ```console
-make
+$ make
+$ make bundle
```

Comment:
- I had a working bundle without `make bundle`. I think this should be part of the install step

Reply:
True. The nomas.app is working with references to the build environment. I'll change this.